packages/Chem/src/tests/menu-tests-chem-space.ts

```diff
@@ -51,8 +51,8 @@ category('top menu chem space', async () => {
     DG.Balloon.closeAll();
     await _testChemSpaceReturnsResult(testSmilesMalformed, 'canonical_smiles');
     try {
-      await awaitCheck(() => (document.querySelector('.d4-balloon-content') as HTMLElement)?.innerText.includes(
-        '2 molecules with indexes 3,10 are possibly malformed and are not included in analysis'), 'cannot find warning balloon', 1000);
+      await awaitCheck(() => document.querySelector('.d4-balloon-content')?.children[0].children[0].innerHTML ===
```

Comment:
I don't think we should test such details, down to the warning text (unless they are critical). DOM might change, messages might change. We have bigger fish to fry :) No reason to change it now of course.